docs/flask_config.md

```diff
@@ -76,3 +76,50 @@ description sources not mentioned in the configuration will be alphabetically pl
+
+##Uneditable Table Descriptions
+Amundsen supports configuring table and column description to be non-editable for selective tables. You may want to make table 
+descriptions non-editable due to various reasons such as table already has table description from source of truth.
+You can define matching rules in `config file` for selecting tables. This configuration is useful as table selection criteria can 
+be company specific which will not directly integrated with Amundsen. 
+You can use different combinations of schema and table name for selecting tables.
+
+Here are some examples when this feature can be used: 
+1. You want to set all tables with a given schema pattern as un-editable.
+2. You want to set all tables with a specific table name pattern in a given schema pattern as un-editable.
+3. You want to set all tables with a given table name pattern as un-editable.
+
+Amundsen has two variables in `config` file which can be used to define match rules:
+1. `UNEDITABLE_SCHEMAS` : Set of schemas where all tables should be un-editable. It takes exact schema name.
+2. `UNEDITABLE_TABLE_DESCRIPTION_MATCH_RULES` : List of MatchRuleObject, where each MatchRuleObject consists of regex for 
+schema name or regex for table name or both.
+
+Purpose of `UNEDITABLE_SCHEMAS` can be fulfilled by `UNEDITABLE_TABLE_DESCRIPTION_MATCH_RULES` but we are keeping both 
+variables for backward compatibility.
+If you want to restrict tables from a given schemas then you can use `UNEDITABLE_SCHEMAS` as follows:
+```python
+UNEDITABLE_SCHEMAS = set(['schema1', 'schema2'])
+```
+After above configuration, all tables in 'schema1' and 'schema2' will have non-editable table and column descriptions.
+
+If you have more complex matching rules you can use `UNEDITABLE_TABLE_DESCRIPTION_MATCH_RULES`. It provides you more flexibility 
+and control as you can create multiple match rules and use regex for matching schema nad table names.
+
+You can configure your match rules in `config file` as follow:
+```python
+UNEDITABLE_TABLE_DESCRIPTION_MATCH_RULES = [
+        # match rule for all table in schema1
+        MatchRuleObject(schema_regex=r"^(schema1)"),
+        # macth rule for all tables in schema2 and schema3
+        MatchRuleObject(schema_regex=r"^(schema2|schema3)"),
+        # match rule for tables in schema4 with table name pattern 'noedit_*'
+        MatchRuleObject(schema_regex=r"^(schema4)", table_name_regex=r"^noedit_([a-zA-Z_0-9]+)"),
+        # match rule for tables in schema5, schema6 and schema7 with table name pattern 'noedit_*'
+        MatchRuleObject(schema_regex=r"^(schema5|schema6|schema7)", table_name_regex=r"^noedit_([a-zA-Z_0-9]+)"),
+        # match rule for all tables with table name pattern 'others_*'
+        MatchRuleObject(table_name_regex=r"^others_([a-zA-Z_0-9]+)")
+    ]
+```
+
+After configuring this, users will not be able to edit table and column descriptions of any table matching above match rules
```

Comment:
nit: I would actually delete the `and will be redirected to the source linked to the table (if any) provided by table metadata service`. The reason is that this in itself is a separate feature/UI decision, and we should limit this section to just describe how to make the table uneditable. What the UI does with it is a separate concern that *could* always change, and so if anything that would be included in some other documentation. Perhaps you could add a full tutorial in our doc site describing the full scope!

Reply:
Okay, got it. I will remove this form here.